app/src/main/java/net/programmierecke/radiodroid2/service/PlayerService.java

```diff
@@ -287,13 +252,39 @@ public long getTransferredBytes() throws RemoteException {
             }
             return 0;
         }
+
+        @Override
+        public long getBufferedSeconds() throws RemoteException {
+            if (radioPlayer != null) {
+                return radioPlayer.getBufferedSeconds();
```

Comment:
Yeah, I'm not sure about all of this... From what I see there even no point in having separate thread for exoplayer since it already does everything in separate thread.

Reply:
Exoplayer just wants every access to be on it's own thread – even simple non-writing inquires. We can ignore this error for now and I can fix it later.